### DISCUSSION TITLE
Items

### DISCUSSION THREAD

**Discussion by alejandquintero** [ismail9k/vue3-carousel#187] [Q&A] [answered]
Hi!

How can i to change the item slide? for example: 4 images in every item

I need than every item be a link for a view, is that possible? how?

Thanks!

**Comment by mursalin3sit** [comment]
**_You can try this way._**

> <template>
    <div class="partner-area">
        <div class="container">
            <div class="partner-slides">
                <carousel
                    :autoplay="3000"
                    :settings='settings'
                    :breakpoints='breakpoints'
                    :wrapAround="true"
                >
                    <slide 
                        v-for="slide in carouselItems" 
                        :key="slide.id"
                    >
                        <div class="partner-item">
                            <img :src="slide.image" alt="image">
                        </div>
                    </slide>
                </carousel>
            </div>
        </div>
    </div>
</template>

><script>
import { defineComponent } from 'vue';
import { Carousel, Slide } from 'vue3-carousel';

import 'vue3-carousel/dist/carousel.css';

export default defineComponent ({
    name: 'Partner',
    components: {
        Carousel,
        Slide,
    },
    data: () => ({
        settings: {
            itemsToShow: 1,
            snapAlign: 'center',
        },
        carouselItems: [
            {
                id: 1,
                image: require('../../assets/images/partner/partner-white1.png'),
            },
            {
                id: 2,
                image: require('../../assets/images/partner/partner-white2.png'),
            },
            {
                id: 3,
                image: require('../../assets/images/partner/partner-white3.png'),
            },
            {
                id: 4,
                image: require('../../assets/images/partner/partner-white4.png'),
            },
        ],
        breakpoints: {
            0: {
                itemsToShow: 1,
                snapAlign: 'center',
			},
            576: {
                itemsToShow: 2,
                snapAlign: 'center',
            },
            768: {
                itemsToShow: 3,
                snapAlign: 'center',
            },
            1200: {
                itemsToShow: 4,
                snapAlign: 'center',
            },
        },
    }),
})
</script>

**Comment by alejandquintero** [comment, marked as answer]
Thanks!
